samples: cellular: deprecate lte_ble_gateway sample (#22111)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Cellular samples
----------------

* :ref:`lte_sensor_gateway` sample:

>   * Sample is removed due to it being unmaintained.

Reviewer: ```suggestion
  * The sample is deprecated due to it being unmaintained.
```

>   * The sample is deprecated due to it being unmaintained.

Reviewer: ```suggestion
  * The sample is no longer maintained and is now deprecated.
```
